core/domain/skill_domain_test.py

```diff
@@ -142,7 +142,9 @@ def test_create_default_skill(self):
             ),
             'language_code': constants.DEFAULT_LANGUAGE_CODE,
             'next_misconception_id': 0,
-            'version': 0
+            'version': 0,
+            'superseding_skill_id': None,
+            'all_questions_merged': None
```

Comment:
Shouldn't this be false? or if it can be None also, add that to the docstring above for class Skill.

Reply:
Oh yeah, it should be False I think.

Reply:
I thought it should be false, but in a test further down, I saw an update where None was actually updated to false. So, got confused.

Reply:
Makes sense, made the changes